src/lobby/GameConnection.cpp

```diff
@@ -47,17 +49,17 @@ void Sapphire::Network::GameConnection::onAccept( const std::string& host, uint1
 }
 
 
-void Sapphire::Network::GameConnection::onDisconnect()
+void Lobby::Network::GameConnection::onDisconnect()
 {
   Logger::debug( "DISCONNECT" );
 }
 
-void Sapphire::Network::GameConnection::onRecv( std::vector< uint8_t >& buffer )
+void Lobby::Network::GameConnection::onRecv( std::vector< uint8_t >& buffer )
 {
   m_packets.insert( std::end( m_packets ), std::begin( buffer ), std::end( buffer ) );
   // This is assumed packet always start with valid FFXIVARR_PACKET_HEADER for now.
-  Packets::FFXIVARR_PACKET_HEADER packetHeader{};
-  const auto headerResult = Packets::getHeader( m_packets, 0, packetHeader );
+  Sapphire::Network::Packets::FFXIVARR_PACKET_HEADER packetHeader{};
```

Comment:
Same here, you are "using" the namespaces at the top. might aswell shorten it here. ( and following usages )